### PR TITLE
chore: update letterbox and similar transforms to use torchvision transforms

### PR DESCRIPTION
**Pull request by gabrielaquiroz**
https://app.clickup.com/t/36001183/PERCEPTION-2500

### CODE REVIEW

File: utils/augmentations.py [hunk 1]
```diff
@@ -382,7 +401,14 @@ def __call__(self, im):  # im = np.array HWC
...
         top, left = (imh - m) // 2, (imw - m) // 2
-        return cv2.resize(im[top:top + m, left:left + m], (self.w, self.h), interpolation=cv2.INTER_LINEAR)
+        # Use torchvision transforms to have parity with production
+        cropped_im = torch.from_numpy(im[top:top + m, left:left + m].transpose(2, 0, 1))
+        resized_im = TF.resize(
+            img=cropped_im,
+            size=[self.h, self.w],
+            interpolation=TF.InterpolationMode.BILINEAR,
+        )
+        return resized_im.numpy().transpose(1, 2, 0)
```
Comment (timwroge) [inline utils/augmentations.py:411]: Could we confirm this is the correct colorspace?

Comment (gabrielaquiroz) [reply utils/augmentations.py:411]: I believe the colorspace is dependent on the hyperparameters you select

**Comment by gabrielaquiroz** [comment]
Closing for now.

File: utils/augmentations.py (hunk 1 above)
Comment (gabrielaquiroz) [reply utils/augmentations.py:411]: This is converting HWC to CHW input

**Pull request closed without merging**